cfme/tests/containers/test_search_bar.py

```diff
@@ -73,4 +73,6 @@ def test_search_bar(provider, soft_assert):
                         'Should not find records, search results: "{}"'
                         .format(search_string, results_row_names))
         finally:
-            search.ensure_no_filter_applied()
+            # search.ensure_no_filter_applied() -> "TimedOutError: Could not do Waiting
+            # for advanced search to open at...", so better for this test to cleanup by:
+            search.normal_search("")
```

Comment:
Hmm, I think we have a function to click the "default" link which should reset all filtering, or is it not applicable here?